examples/04_pong/systems/winner.rs

```diff
@@ -34,10 +37,20 @@ impl<'s> System<'s> for WinnerSystem {
             let did_hit = if ball_x <= ball.radius {
                 // Right player scored on the left side.
                 score_board.score_right += 1;
+                for (transform, text) in (&ui_transform, &mut text).join() {
+                    if "P2" == transform.id {
```

Comment:
Entities should be used for this. While having a name for entities (which I btw. planned to implement in Specs: you would be able to just do `world.create_entity().name("Blah")` then) is nice, I don't think we should recommend it for component identification purposes.

Reply:
I guess I like the getElementById from JavaScript and I was trying to mimic that here.  It's nice being able to just keep things in your head and request for static values rather than having to store the entity, but storing the entity is also more efficient, so I'll go with that.